islands/pipeline.tsx

```diff
@@ -223,6 +226,13 @@ const PipelineDetail = (
         dragOrder: data.steps.length,
       }]],
     });
+    setOpen([...open, data.steps.length]);
+    setTimeout(() => initFlowbite(), 1000);
```

Comment:
Pretty gross hack here to re-initialize flowbite for dynamically added components. Flowbite doesn't work great in this stack.